api/client/src/main/java/org/projectnessie/client/auth/oauth2/OAuth2ClientConfig.java

```diff
@@ -200,30 +219,64 @@ private static HttpClientException genericError(Status status) {
         "OAuth2 server replied with HTTP status code: " + status.getCode());
   }
 
+  private static void check(
+      List<String> violations, String paramKey, boolean cond, String msg, Object... args) {
+    if (!cond) {
+      if (args.length == 0) {
+        violations.add(msg + " (" + paramKey + ")");
```

Comment:
nit: we could probably call `format()` with an empty arg list and avoid this `if`